docs/deploying_apps/deploying_node_js.md

```diff
@@ -2,46 +2,43 @@ This section covers how to deploy a basic Node.js application to Government PaaS
 
 Note that the only database service currently supported by PaaS is PostgreSQL. If your Node.js app requires a database, it must be able to work with PostgreSQL.
 
-Your app will need to parse the ``VCAP_SERVICES`` environment variable to get details of any backing services, such as PostgreSQL. See the section [Node.js and backing services](#nodejs-and-backing-services) below.
+These instructions assume you have already carried out the setup process explained in the [Quick Setup Guide](/getting_started/quick_setup_guide) section.
 
-These steps assume you have already carried out the setup process explained in the [Quick Setup Guide](/getting_started/quick_setup_guide) section.
-
-
-1. Check out your Node.js app to a local folder.
-2. [Exclude files ignored by Git](/deploying_apps/excluding_files/).
-3. Create a ``manifest.yml`` file:
+1. Check out your app to a local folder.
+1. [Exclude files ignored by Git](/deploying_apps/excluding_files/).
+1. Add a ``manifest.yml`` file:
 
         ---
         applications:
         - name: my-node-app
           memory: 256M
           buildpack: nodejs_buildpack
 
-3. Include an npm ``package.json`` file to specify dependencies. The file should also specify a `start` command used to launch the app.
+    See [Deploying with Application Manifests](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html) on the Cloud Foundry site for details.
+
+1. Make sure you have an npm ``package.json`` file to specify dependencies. The file should specify a `start` command used to launch the app, unless this is provided in a separate Procfile.
```

Comment:
Have removed "to specify dependencies" and added below the JSON example: 

"You must provide a ``package.json`` file even if your app has no dependencies."